integration/plc-integration/src/main/java/com/hbc/pms/plc/io/IoDetailsRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.hbc.pms.plc.io;
+
+import java.util.List;
+
+public interface IoDetailsRepository {
```

Comment:
Please use IIoDetailsRepository

Reply:
@oHTGo We don't do that here in Java Spring Project. We shouldn't use "I" for interface and "Impl" in our Spring project.